### DISCUSSION TITLE
Anker Smart Meter

### DISCUSSION THREAD

**Discussion by TOlaf** [thomluther/ha-anker-solix#83] [Ideas]
I can see my Anker Smart Meter in the device list of the integration, but there are no entities. 
It would be nice to have an entity for the actual kW.

**Comment by thomluther** [comment]
Neither the Solarbank nor the Smartmeter are actually supported by the Integration.
Base work on the Api library is necessary first by the community. You are welcome to contribute, See [Api library issues](https://github.com/thomluther/anker-solix-api/issues)

**Comment by Bollo2105** [comment]
Hey @thomluther, first thank you for this amazing integration.
As I am pretty new with the Anker integration to HA. I read in your release notes, that the Smart Meeter is also integrated in Reles 2.0.0 .
Unfortunately, my smart Meeter will not be detected by the API. Do you have some ideas where to search for it, to get in in?

**Reply by thomluther** [reply]
Have you installed version 2.0.0?
If your smart meter is in the power system, it should also be found by the integration.
You can run the Get System information services to see what the cloud responds for your system.
